src/patternfly/components/Dropdown/dropdown-toggle-check.hbs

```diff
@@ -0,0 +1,7 @@
+<label class="pf-c-dropdown__toggle-check{{#if dropdown-toggle-check--modifier}} {{dropdown-toggle-check--modifier}}{{/if}}"
+  for="{{dropdown-toggle-check--id}}"
+  {{#if dropdown-toggle-check--attribute}}
+	  {{{dropdown-toggle-check--attribute}}}
+  {{/if}}>
+  <input type="checkbox" id="{{dropdown-toggle-check--id}}">
+</label>
```

Comment:
I like that you wrap the `<input>` with a `<label>` to maximize the target area.

This use case is comparable to what we have for the Table component, which uses the following:
```
<td class="pf-c-table__check">
  <input class="pf-c-check__input" type="checkbox" name="check-all" aria-label="Select all rows">
</td>
```
The checkbox in this Table example includes `aria-label` which we also need in the new Split Button example. I think the same label "Select all rows" is applicable here for consistency, since the behavior is the same. I'm adding @stacymcauliffe just to confirm if this is the label we want to standardize on for this control, or if we want to make it more generic like "Select all" or "Select all items" since I think this control you're adding would be used for tables, list views, and card views.

Also, when we chatted last week about the this split button example, you mentioned that the class `pf-c-check__input` shouldn't be applied in that example. I captured that in issue #1271. Please update if I captured this incorrectly. And if we decide that something other than "Select all rows" is the label we want, we can note that in the same issue.